.gitignore

```diff
@@ -1,5 +1,7 @@
 .vscode/
-wurst/war3map.j
-objectEditingOutput/*
-compiled.j.txt
+/_build
```

Comment:
Can I suggest to make an issue and we can agree what to do in this PR? It will save us both time.

Reply:
I won't take removing `_build` yet - but in future, yes.

Reply:
As i said, this is just "how it should be" and you don't have to merge if you wanna stick messy for now.

Reply:
@Frotty are you saying you won't edit this PR to match my requirements?